### PR TITLE
851 -  [Chart] Allow to use stroke line without dashes

### LINKED ISSUE #851: [Chart]  Allow the use of stroke line without dashes
## Component to amend
Chart - Reference Area

## Visual
As you can see in P34 and P18, the stroke line is without dashes

Screenshot from 2024-09-03 10-51-06

You need to change [this line](https://github.com/laboratoriobridge/bold/blob/main/src/components/Chart/renderSeries.tsx#L54)

### PR DESCRIPTION
**Pull request by AndresKindel**
closes  #851

### CODE REVIEW

**Review by wesleydecezere — DISMISSED** [review]

**Review by GuilhermePrandi — DISMISSED** [review]

**Review by wagnerbsantos — APPROVED** [review]

File: src/components/Chart/Chart.stories.tsx [hunk 1]
```diff
@@ -135,6 +136,7 @@ const referenceAreas: ReferenceArea<number>[] = [
     ],
     color: '#ffeed6',
     tickColor: '#b58b00',
+    dashed: true,
```
Comment (zDudaHang) [inline src/components/Chart/Chart.stories.tsx:139]:
```suggestion
    dashed: false,
```
I think it's important to show all options at storybook

**Review by zDudaHang — APPROVED** [review]

File: src/components/Chart/model.ts [hunk 2]
```diff
@@ -90,6 +90,7 @@ export interface ReferenceArea<XDomain> {
   tickColor?: string
   stroke?: boolean
   strokeColor?: string
+  dashed?: boolean
```
Comment (Andrevmatias) [inline src/components/Chart/model.ts:93]:
```suggestion
  strokeType?: 'line' | 'dashed'
```
It's more explicit and scalable

**Review by Andrevmatias — CHANGES_REQUESTED** [review]
Great! 

Just take into account that this component must follow the bold rules for [data visualization](https://bold.bridge.ufsc.br/en/design-guidelines/data-visualization/usage). I think it would be really good to evolve the specification in the bold's site/prototyte to better define the uses and resources of the reference area.

**Review by wagnerbsantos — APPROVED** [review]

**Review by Andrevmatias — APPROVED** [review]
Great! :blush:

**Pull request merged**